data/mships.txt

```diff
@@ -4200,7 +4200,7 @@ ship "PT-60 Gallant"
 	explode "large explosion" 35
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
-	description `Solomons Voidworks protected tanker ship for shipping crude oil and other petroleum products from and across the southern space. Heavily armoured and armed, the Gallant provides excellent protection against any pirates it might come across. While difficulty of handling bulk liquid cargo deter many pirates from stealing it, report of raid on oil tanker is not unheard of. It is suspected that they either have no other targets or it was an attempt to capture the entire ship.`
+	description `A Solomon Voidworks protected tanker ship for shipping crude oil and other petroleum products from and across the southern part of human space. Heavily armoured and armed, the Gallant provides excellent protection against any pirates it might come across. While the difficulty of handling bulk liquid cargo deters many pirates from stealing it, reports of raid on oil tankers are not unheard of. It is suspected that they either have no other targets or they were attempts to capture the entire ship.`
```

Comment:
```suggestion
	description `A Solomons Voidworks protected tanker ship for shipping crude oil and other petroleum products from and across the Southern space. Heavily armoured and armed, the Gallant provides excellent protection against any pirates it might come across. While the difficulty of handling bulk liquid cargo deters many pirates from stealing it, reports of raid on oil tankers are not unheard of. It is suspected that they either have no other targets or they were attempts to capture the entire ship.`
```
"Solomons Voidworks" is the name of the shipyard. Southern space is what people call that area of space and this ship is likely spoken in context of human space already.